ClosedXML/Excel/Cells/XLCell.cs

```diff
@@ -700,6 +700,7 @@ public IXLTable InsertTable<T>(IEnumerable<T> data, String tableName, Boolean cr
 
         public IXLTable InsertTable<T>(IEnumerable<T> data, String tableName, Boolean createTable, Boolean addHeadings, Boolean transpose)
         {
+
```

Comment:
Please remove this whitespace change too.

Reply:
Done.